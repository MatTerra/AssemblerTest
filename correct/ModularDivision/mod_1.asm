mod_1: begin
section data
result: space
section text
n: extern
mod_2: extern
public finish
public divide
input n
input n
jmp mod_2
divide: load n
div n
store result
output result
finish: stop
end
