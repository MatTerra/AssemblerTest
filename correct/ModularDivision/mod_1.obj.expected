12 22 12 22 05 16 10 22 04 22 11 15 13 15 14 00 10 22 08 14 05 06 00 